swap/src/protocol/alice/swap.rs

```diff
@@ -78,20 +78,31 @@ async fn run_until_internal(
         match state {
             AliceState::Started { state3 } => {
                 timeout(
```

Comment:
I think this timeout should also lead to `SavelyAborted` - or should the swap remain in `Started` forever in the database?

Reply:
yep you are right